plutus-core/executables/plutus/Mode/PrintBuiltins.hs

```diff
@@ -0,0 +1,81 @@
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE ViewPatterns #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE DataKinds           #-}
+module Mode.PrintBuiltins
+    ( runPrintBuiltins
+    ) where
+
+import PlutusPrelude
+import PlutusCore qualified as PLC
+import PlutusCore.Builtin qualified as PLC
+import PlutusCore.Normalize (normalizeType)
+
+import Prettyprinter qualified as PP
+import Text.Printf
+import Data.Proxy (Proxy (..))
+import Data.List (intercalate)
+import GHC.TypeLits (symbolVal)
+
+runPrintBuiltins :: IO ()
+runPrintBuiltins = portedImpl
+
+-- following is PORTED from the older executable
```

Comment:
Do we need this comment for something?

Reply:
Not really, it was for you, so you don't have to review the following code :D